src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 1px 3px 5px #{$header-shadow-color};
+}
+
+.logo-img {
+  display: block;
+}
+
+.header__nav {
```

Comment:
Yep, @DionStrzh is correct. Better to have `nav` as separate block.

Reply:
Do you mean like this ?

```
  <body>
    <header class="header">
      <a href="https://d-hubych.github.io/layout_moyo-header/">
        <img
          class="logo-img"
          src="./images/logo.png"
          alt="logo"
        >
      </a>

      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#apple">Apple</a>
          </li>
```